packages/slice-machine/src/legacy/lib/builders/common/Zone/index.jsx

```diff
@@ -198,22 +198,6 @@ const Zone = ({
               testId={testId}
               // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
               isRepeatableCustomType={isRepeatableCustomType}
-              newField={
```

Comment:
❓ Does this prop need to be removed from `Card` now?

Reply:
Not needed, removed ✅